test/without.js

```diff
@@ -17,15 +19,10 @@ describe('without', function() {
   });
 
   it('has R.equals semantics', function() {
-    function Just(x) { this.value = x; }
-    Just.prototype.equals = function(x) {
-      return x instanceof Just && R.equals(x.value, this.value);
-    };
-
```

Comment:
It's definitely a nice thing to remove these types from the individual test.  But the win is not that big, as we could simply place them in `test/shared` (and probably already should have.)

Reply:
I'm not sure what you're suggesting here. Would you prefer to duplicate the code for Sanctuary's Maybe type in the Ramda test suite? I worry that the Ramda version wouldn't be kept up to date as additions are made to the Fantasy Land spec.

Reply:
Actually, it turns out that there's already a `Maybe` there.  These tests should have referred to it.  But I don't at all mind switching to using Sanctuary's `Maybe` instead... except that if we try to track latest, we could be burned by tests breaking without an (internal) change.